lib/rds_sql_server/rds.rb

```diff
@@ -37,19 +35,7 @@ def self.timeout
   end
 
   def initialize
-    # if environment is set up correctly, this can be
-    # self.client = Aws::RDS::Client.new
-    if SECRET_ACCESS_KEY.present? && SECRET_ACCESS_KEY != 'unknown'
-      self.client = Aws::RDS::Client.new({
-                                           region: REGION,
-                                           access_key_id: ACCESS_KEY_ID,
-                                           secret_access_key: SECRET_ACCESS_KEY,
-                                         })
-    else
-      self.client = Aws::RDS::Client.new({
-                                           region: REGION,
-                                         })
-    end
+    self.client = Aws::RDS::Client.new
```

Comment:
We should abandon long-lived keys, and spot checking a few roles in ECS, it looks like these keys aren't serving a purpose. Let me know if there's some reason to keep the ability to inject different static creds into this code or if I'm mistaken.

Reply:
Hmm, we might still need the region (see line 50).  I'm pretty sure we fixed up the roles, but we should definitely test that.